sys/include/net/nanocoap.h

```diff
@@ -224,6 +225,15 @@ extern "C" {
 #define COAP_BLOCKWISE_SZX_MAX  (7)
 /** @} */
 
+/**
+ * @name coap_opt_finish() flag parameter values
```

Comment:
A small description of the purpose of these flags would be helpful here.

Reply:
Will do.